lib/movable_ink/aws/pager_duty_alert.rb

```diff
@@ -0,0 +1,50 @@
+require 'json'
+require 'time'
+require 'aws-sdk-sns'
+
+class PagerDutyAlert
+  def initialize(
+    client:,
+    source:,
+    summary:,
+    dedup_key:,
+    topic_arn:,
+    links: [],
+    custom_details: {}
+  )
+    @client = client
+    @source = source
+    @summary = summary
+    @links = links
+    @dedup_key = dedup_key
+    @custom_details = custom_details
+    @topic_arn = topic_arn
+  end
+
+  def message_hash
+    {
+      pagerduty: {
+        event_action: 'trigger',
+        payload: {
+          source: @source,
+          summary: @summary,
+          timestamp: Time.now.utc.iso8601,
+          severity: 'error',
+          component: @source,
+          group: @source,
+          custom_details: @custom_details,
+        },
+        dedup_key: @dedup_key,
+        links: @links,
+      }
+    }
+  end
+
+  def publish
+    @client.publish({
```

Comment:
This was previously wrapped in `run_with_backoff`.  Do you not want to retain that?

Reply:
I felt like it was enough for `run_with_backoff` to happen in the caller, since this is a standalone class we don't have access to that method. Are you thinking it would be better to have it in this method?

Reply:
Is there a reason for it to be a standalone class rather than moduled within `MovableInk::AWS` that I'm not thinking of?